serverless.yml

```diff
@@ -5,16 +5,24 @@ provider:
   region: us-east-1
   timeout: 10
   memorySize: 192
+  lambdaHashingVersion: 20201221
   stage: ${opt:stage, 'dev'}
-  runtime: nodejs12.x
+  runtime: nodejs14.x
+
+package:
+  patterns:
+    - "!*"
+    - "!*/**"
+    - "dist/**"
+    - "github-app-probot-home-assistant-key.pem"
```

Comment:
I'm not too sure about this.
Our previous exclude did nothing.
With this only the dist dir and the certificate will be packaged, AFAIK that is the only things we need.